python/ray/data/_internal/stats.py

```diff
@@ -34,21 +34,38 @@ class Timer:
 
     def __init__(self):
         self._value: float = 0
+        self._min: float = float("inf")
+        self._max: float = 0
+        self._total_count: float = 0
 
     @contextmanager
     def timer(self) -> None:
-        time_start = time.perf_counter()
+        time_start = time.thread_time()
```

Comment:
Hmm, why the change to thread (CPU) time? That can be unexpected when reported naively without the wall time context.

Reply:
reverted back